pages/common/bower.md

```diff
@@ -0,0 +1,32 @@
+# bower
+
+> A package manager optimized for the front-end web development.
```

Comment:
Just "front-end development" or "web development". "Front-end web development" is a tautology.